com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderProgramBuilder.java

```diff
@@ -103,24 +93,30 @@ static public String compileGLSL(String shaderSource, ES2ToES3Converter.ShaderTy
             }
         }
 
-        // Write our directives.
         if (directiveLineCount != 0) {
             writer.println();
         }
 
-        if (shaderLanguage == ShaderDesc.Language.LANGUAGE_GLES2) {
+        boolean gles =  shaderLanguage == ShaderDesc.Language.LANGUAGE_GLES_SM100 ||
+                        shaderLanguage == ShaderDesc.Language.LANGUAGE_GLES_SM300;
+
+        boolean gles3Standard = shaderLanguage == ShaderDesc.Language.LANGUAGE_GLSL_SM140 ||
+                                shaderLanguage == ShaderDesc.Language.LANGUAGE_GLES_SM300;
```

Comment:
Removed the "platform" flags in favor of the more technically correct shader model enum.